entity_translation

```diff
@@ -0,0 +1 @@
+Subproject commit 99f7b6ba722a943a5a95e411894608ca7230d5e7
```

Comment:
I think this got added by mistake since I'm also working on a patch for another issue. Is there a way I can delete this @angaither ?